src/game/player.ts

```diff
@@ -148,6 +149,9 @@ export class Player extends GameObject {
             eventSpawnProjectile({
                 position: this.collider.getPosition().getSum(
                     // Spawn projectile right in front of the player so it doesn't collide with them
+                    // There is some rounding error that I cannot reproduce in unit tests, but when
+                    // you turbo and fire, then your own projectile will damage you, unless the +1
+                    // is in the expression below.
```

Comment:
which +1 ?

Reply:
The +1 which miraculously was not commited

Reply:
Hm, I don't like it, but it doesn't seem to affect anything, so I'll let it slide :)